docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -335,7 +335,7 @@ So now we have a working gateway with DNS and TLS configured. Let place this gat
 1. In `T2` window execute the following to see the gateway on the workload-1 cluster:
 
     ```bash
-    kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/local/kube/workload1.yaml && export KUBECONFIG=$(pwd)/local/kube/workload1.yaml
+    kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/workload1.yaml && export KUBECONFIG=$(pwd)/workload1.yaml
```

Comment:
this doesn't look right, why have the kubeconfig files moved to the root directory?

Reply:
root directory? You mean of the repo? It is only for the walkthrough and the local/kube dir may not exist locally

Reply:
I can remove it if you want. Wasn't really a part of this task just thought I would pick up while it was in my head